ssl/statem/extensions.c

```diff
@@ -372,6 +376,19 @@ static const EXTENSION_DEFINITION ext_defs[] = {
         tls_construct_certificate_authorities,
         tls_construct_certificate_authorities, NULL,
     },
+#ifndef OPENSSL_NO_QUIC
+    {
+        TLSEXT_TYPE_quic_transport_parameters,
+        SSL_EXT_CLIENT_HELLO | SSL_EXT_TLS1_3_SERVER_HELLO
```

Comment:
I think it should be SSL_EXT_TLS1_3_ENCRYPTED_EXTENSIONS instead of SSL_EXT_TLS1_3_SERVER_HELLO.

Reply:
Yup... I misread the BoringSSL code, the function name implies it's being added to the ServerHello